src/main/java/baritone/pathing/movement/CalculationContext.java

```diff
@@ -150,7 +152,7 @@ public double costOfPlacingAt(int x, int y, int z, IBlockState current) {
     }
 
     public double breakCostMultiplierAt(int x, int y, int z, IBlockState current) {
-        if (!allowBreak) {
+        if (!allowBreak || baritone.getPathingBehavior().isCloseToHome()) {
```

Comment:
same here
either cached or block dependant

Reply:
Changed in latest commit